tests/test_utils/__init__.py

```diff
@@ -78,25 +79,29 @@ def setup_awaitable_errors() -> Callable[[], None]:
     unraisable_exceptions = []
     orig_unraisablehook = sys.unraisablehook
 
-    def unraisablehook(unraisable):
+    def unraisablehook(unraisable: sys.UnraisableHookArgs) -> None:
```

Comment:
huh, is that a type hint in the stdlib?

Reply:
It is a type / class, not really a type hint?

Reply:
Okay fair, but I don't think it exists at runtime:

```
$ python -c 'import sys; sys.UnraisableHookArgs'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: module 'sys' has no attribute 'UnraisableHookArgs'. Did you mean: 'unraisablehook'?
```

I'm wondering if this might have come from typeshed(?)

Reply:
Ahh yes, see https://github.com/python/typeshed/blob/cf6ff75952c21e4b5de72b2d0d3d0f0cbb14a563/stdlib/sys.pyi#L300-L311

you might have to enclose it in quotes, to stop python looking up the nonexistant attribute at runtime?

Reply:
Ah, I did not realize that typeshed injects things like that.